thirdparty/runtime_environments.go

```diff
@@ -58,3 +66,42 @@ func (c *RuntimeEnvironmentsClient) getImageNames(ctx context.Context) ([]string
 	}
 	return filteredImages, nil
 }
+
+// PackageFilterOptions represents the filtering arguments, each of which is optional.
+type PackageFilterOptions struct {
+	Page    int
+	Limit   int
+	Name    string
+	Manager string
```

Comment:
could be worth adding a comment for name and manager, since these aren't standard